packages/legacy/src/client/helpers.ts

```diff
@@ -167,6 +181,14 @@ export const ovirtUrlToHostname = (url: string): string => {
 export const ovirtHostnameToUrl = (hostname: string): string =>
   `https://${hostname}/ovirt-engine/api`;
 
+export const openstackUrlToHostname = (url: string): string => {
```

Comment:
please add unit tests

Reply:
@sgratch , if they are small add in this PR, o/w if this will take some time, can you split it and do the unit tests in a folowup ?

Reply:
I'll add unit tests in a followup PR after approving the required fields in this PR